frontend/src/App.vue

```diff
@@ -97,7 +97,7 @@ export default {
 }
 
 .constrained {
-  max-width: 1024px !important;
+  max-width: 73rem !important;
```

Comment:
J'ai créé une carte dans le jardin (backlog technique) pour ne pas oublier de se pencher sur le sujet, notamment pour les layouts avec colonne latérale
```suggestion
  max-width: 1024px !important;
```